src/RA_AchievementSet.cpp

```diff
@@ -353,27 +291,15 @@ BOOL AchievementSet::FetchFromWebBlocking(ra::GameID nGameID)
         doc["Success"].GetBool() &&
         doc.HasMember("PatchData"))
     {
-        std::wstring sAchSetFileName;
-        {
-            std::wostringstream oss;
-            oss << g_sHomeDir << RA_DIR_DATA << nGameID << L".txt";
-            sAchSetFileName = oss.str();
-        }
-
-        std::ofstream ofile{ sAchSetFileName };
-        if (!ofile.is_open())
+        auto& pLocalStorage = ra::services::ServiceLocator::GetMutable<ra::services::ILocalStorage>();
+        auto pData = pLocalStorage.WriteText(ra::services::StorageItemType::GameData, std::to_wstring(static_cast<unsigned int>(nGameID)));
```

Comment:
Not a big deal but the `static_cast<unsigned int>` is redundant. `ra::GameID` _uses_ the`std::size_t` definition, if `std::to_wstring` works for `std::size_t` it should work for `ra::GameID`.
With a `using `(type alias) the alias is identical to the original type, not different.

Reply:
Changed. Though I was sure I had to put that in for some reason - it's not something I would have thought to do without a warning/error.